src/sentry/web/api.py

```diff
@@ -307,9 +344,6 @@ def post(self, request, **kwargs):
             # bubble up as an APIError.
             data = None
 
-        if pubsub is not None and data is not None:
-            pubsub.publish('requests', data)
```

Comment:
Looks like this is not in use anymore, but would be great if someone confirmed this.
cc @alex-hofsteede